server/commands.py

```diff
@@ -40,14 +40,12 @@ def choices(self):
 @dataclass(frozen=True)
 class MessageDelayCommand(GenericMessageCommand):
     delay_ms: int = 1000
-    expects_user_input = False
```

Comment:
Why are we removing the flags here? This might cause issues with the frontend, lmk